web/src/click_handlers.js

```diff
@@ -726,7 +726,7 @@ export function initialize() {
         "click",
         ".private_messages_container.zoom-out #private_messages_section_header",
         (e) => {
-            if (e.target.classList.value === "fa fa-align-right") {
+            if ($(e.target).closest("#show_all_private_messages").length > 0) {
```

Comment:
I don't think `length > 0` makes sense here; it's an ID, so there should only ever be 0 or 1 of them (if you're seeing something different, though, we should address that separately). In that event, the condition I'd be checking is `length === 1`.

Reply:
Thanks @karlstolley!! I verified this and you're right it would just be either 0 or 1 so to tighter this check, I have modified this to check `length === 1`.

I also verified that the functionality for all cases works as expected after this change.
cc: @amanagr